### PR TITLE
Renaming various test .infs to match convention

### PR DESCRIPTION
**Pull request by kenlautner**
## Description

For our various test apps we have a naming convention where we include TestApp in the file name and in the module name in the .inf file.  This PR renames various test .infs to match the TestApp convention.

- [ ] Impacts functionality?
  - **Functionality** - Does the change ultimately impact how firmware functions?
  - Examples: Add a new library, publish a new PPI, update an algorithm, ...
- [ ] Impacts security?
  - **Security** - Does the change have a direct security impact on an application,
    flow, or firmware?
  - Examples: Crypto algorithm change, buffer overflow fix, parameter
    validation improvement, ...
- [ ] Breaking change?
  - **Breaking change** - Will anyone consuming this change experience a break
    in build or boot behavior?
  - Examples: Add a new library class, move a module to a different repo, call
    a function in a new library class in a pre-existing module, ...
- [ ] Includes tests?
  - **Tests** - Does the change include any explicit test code?
  - Examples: Unit tests, integration tests, robot tests, ...
- [ ] Includes documentation?
  - **Documentation** - Does the change contain explicit documentation additions
    outside direct code modifications (and comments)?
  - Examples: Update readme file, add feature readme file, link to documentation
    on an a separate Web page, ...

## How This Was Tested

Passing CI

## Integration Instructions

N/A

### CODE REVIEW

**Comment by makubacki** [comment]
I know this might not be ready yet, but please try to make the PR title and description a little more accurate to the change than before.

File: CryptoPkg/CryptoPkg.dsc [hunk 1]
```diff
@@ -67,6 +67,7 @@
   # Add support for stack protector
   # NULL|MdePkg/Library/BaseStackCheckLib/BaseStackCheckLib.inf # MU_CHANGE: Use Project Mu StackCheckLib
 
+
```
Comment (os-d) [inline CryptoPkg/CryptoPkg.dsc]: drop

File: MdePkg/Test/UnitTest/Library/BaseLib/BaseLibUnitTestApp.inf [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
-  MdePkg/MdePkg.dec
-
-[LibraryClasses]
-  BaseLib
-  BaseMemoryLib
-  UefiApplicationEntryPoint
-  DebugLib
-  UnitTestLib
+## @file
+# Unit tests of Base64 conversion APIs in BaseLib that are run from UEFI Shell.
```
Comment (os-d) [inline MdePkg/Test/UnitTest/Library/BaseLib/BaseLibUnitTestApp.inf]: Did line endings change here? Can you ensure they are CRLF?

File: MdePkg/Test/UnitTest/Library/BaseSafeIntLib/TestBaseSafeIntLibTestApp.inf [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
-[LibraryClasses]
-  UefiApplicationEntryPoint
-  BaseLib
-  DebugLib
-  SafeIntLib
-  UnitTestLib
+## @file
+# UEFI Shell based Application that Unit Tests the SafeIntLib
+#
+# Copyright (c) Microsoft Corporation.<BR>
```
Comment (os-d) [inline MdePkg/Test/UnitTest/Library/BaseSafeIntLib/TestBaseSafeIntLibTestApp.inf]: Did line endings change here? Can you ensure they are CRLF?

**Review by os-d — APPROVED** [review]

File: MdePkg/Test/UnitTest/Library/BaseLib/BaseLibUnitTestApp.inf (hunk 2 above)
Comment (kenlautner) [reply MdePkg/Test/UnitTest/Library/BaseLib/BaseLibUnitTestApp.inf]: Confirmed that they're CRLF.

File: MdePkg/Test/UnitTest/Library/BaseSafeIntLib/TestBaseSafeIntLibTestApp.inf (hunk 3 above)
Comment (kenlautner) [reply MdePkg/Test/UnitTest/Library/BaseSafeIntLib/TestBaseSafeIntLibTestApp.inf]: Confirmed that they're CRLF.

File: CryptoPkg/CryptoPkg.dsc (hunk 1 above)
Comment (kenlautner) [reply CryptoPkg/CryptoPkg.dsc]: Fixed.

File: MdeModulePkg/Application/SmiHandlerProfileInfo/SmiHandlerProfileAuditTestApp.inf
Comment (makubacki) [inline MdeModulePkg/Application/SmiHandlerProfileInfo/SmiHandlerProfileAuditTestApp.inf:1]: For the benefit of others in the future, can you please briefly explain the convention in the PR description? Inevitably, someone might look at the changes and think "what is the convention here"?

Comment (makubacki) [reply MdeModulePkg/Application/SmiHandlerProfileInfo/SmiHandlerProfileAuditTestApp.inf:1]: Just make what's there a little more explicit.

File: MdePkg/Test/UnitTest/Library/BaseLib/BaseLibUnitTestApp.inf (hunk 2 above)
Comment (makubacki) [reply MdePkg/Test/UnitTest/Library/BaseLib/BaseLibUnitTestApp.inf]: I pulled your branch locally and see LF. Can you please double check your autocrlf settings and try pulling directly from the remote branch?

image

**Review by makubacki — APPROVED** [review]

File: MdePkg/Test/UnitTest/Library/BaseLib/BaseLibUnitTestApp.inf (hunk 2 above)
Comment (kenlautner) [reply MdePkg/Test/UnitTest/Library/BaseLib/BaseLibUnitTestApp.inf]: I had autocrlf set to true for my local repo.  Setting that to false allows me to see the line endings be LF.  I pushed a fix.

File: MdePkg/Test/UnitTest/Library/BaseSafeIntLib/TestBaseSafeIntLibTestApp.inf (hunk 3 above)
Comment (kenlautner) [reply MdePkg/Test/UnitTest/Library/BaseSafeIntLib/TestBaseSafeIntLibTestApp.inf]: See above comment.  The line endings were LF so I changed them to CRLF.

**Pull request merged**